lib/ansible/modules/network/ovs/openvswitch_bridge.py

```diff
@@ -140,6 +140,12 @@ def map_obj_to_commands(want, have, module):
                                 or want['external_ids'][k] != have['external_ids'][k]):
                             command += " " + k + " " + v
                             commands.append(command)
+
+            if want['vlan'] and want['vlan'] != have['vlan']:
```

Comment:
I'm not sure what you check with `if want['vlan']` but it seems wrong without any explanation...

Reply:
there's no explanation in the rest of the module either, so I think documenting the module is outside of the scope of this patch. In any case it basically checks that it's getting a vlan from args and such vlan is not the same as the one already existing on the bridge

Reply:
The `if want['vlan']` test isn't needed. That value will default to `None` per the `argument_spec`.

Reply:
from a main pythonic perspective it won't be needed, but if we remove that part it might be conflicting with some test config fixtures where want['vlan'] might be None and have['vlan'] could be a fixture, such as 0. Having this check prevents a false check there.